.gitmodules

```diff
@@ -58,3 +58,6 @@
 [submodule "src/rapidjson"]
 	path = src/rapidjson
 	url = https://github.com/ceph/rapidjson
+[submodule "src/common/cxx_function"]
+	path = src/common/cxx_function
+	url = git://github.com/potswa/cxx_function
```

Comment:
can we use the https url ? git protocol is not so friendly for those behind the firewall, thanks!